aspnetcore/signalr/troubleshoot.md

```diff
@@ -0,0 +1,91 @@
+---
+title: ASP.NET Core SignalR connection troubleshooting
+author: 
+description: ASP.NET Core SignalR connection troubleshooting.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: 
+ms.custom: mvc
+ms.date: 04/08/2020
+no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: signalr/troubleshoot
+---
+# Troubleshoot connection errors
+
+This section provides help with errors that can occur when trying to establish a connection to a ASP.NET Core SignalR hub.
+
+### Response code 404
+
+When using WebSockets and `skipNegotiation = true`
+```log
+WebSocket connection to 'wss://xxx/HubName' failed: Error during WebSocket handshake: Unexpected response code: 404
+```
+
+* If the connection uses the ID and takes too long to send a request to the server after the negotiate, the server:
+
+  * Deletes the ID.
+  * Returns a 404.
```

Comment:
This is probably the least common cause of a 404, don't you think? I would probably order the bullet points with bad path first, no sticky session second and this last.